### PR TITLE
[DEVHAS-292] Use tini to manage zombie processes

### PR DESCRIPTION
**Pull request by kim-tsao**
### What does this PR do?:

Install the tini package and update our entrypoint to run under tini 
Reference: https://github.com/krallin/tini#more

### Which issue(s)/story(ies) does this PR fixes:

https://issues.redhat.com/browse/RHTAPBUGS-292

### PR acceptance criteria:

- [ ] Unit/Functional tests

  

- [ ] Documentation 

   

- [ ] Client Impact

### CODE REVIEW

**Review by elsony — APPROVED** [review]
The change looks ok and straight forward. The main thing to check on this PR is to fully test the change out after switching to tini to make sure there is no side effect on the change.

**Review by maysunfaisal — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +33,16 @@ COPY --from=builder /workspace/manager .
 COPY appdata.gitconfig /.gitconfig
 RUN chgrp -R 0 /.gitconfig && chmod -R g=u /.gitconfig
 
+# build tini and install to manage zombie proceses
+RUN git clone --branch v0.19.0 https://github.com/krallin/tini /tini
```
Comment (johnmcollier) [inline Dockerfile:37]: Perhaps this should be moved to its own step? Otherwise we'll have to rebuild tini each time we modify HAS.

**Review by johnmcollier — COMMENTED** [review]
Just one suggestion

File: Dockerfile (hunk 1 above)
Comment (kim-tsao) [reply Dockerfile:37]: yes, we will need to avoid rebuilding

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**